tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -137,6 +138,9 @@ def setUp(self):
         logger.info('Test run resource prefix: %s, suffix: %s',
                     self.resource_prefix, self.resource_suffix)
 
+        self.config_scope = self.td.make_resource_name(
+            xds_flags.CONFIG_SCOPE.value)
+
```

Comment:
yea, this is an error. self.td is set on the next line.

Reply:
Saved me twenty minutes on the test run. But this means that I cannot use `make_resource_name`, since `initTrafficDirectorManager` depends on `self.config_scope`. Switching back to the original approach.

Reply:
I see, that's a bummer. Maybe you could provide it to `create_mesh` as an argument (since it's the only place where it's used inside `TrafficDirectorAppNetManager`, and not require it in the constructor?

Reply:
Scope is currently only used by the one `Mesh`, but it will be shared by multiple resources in the future. I think it should probably stay as a member of the class.